.release-please-manifest.json

```diff
@@ -15,5 +15,5 @@
   "packages/sdk-node": "22.2.0",
   "packages/sdk-rtl": "21.3.1",
   "packages/wholly-sheet": "0.5.25",
-  "python": "22.2.0"
+  "python": "22.3.0"
```

Comment:
we'll need a `release-as: 22.2.1` for python in release-please-config.json if we end up doing a patch release before 22.4. I think there are other patches coming (https://github.com/looker-open-source/sdk-codegen/pull/1006 ?) that might warrant a top-level `release-as: 22.2.1` key. cc @drstrangelooker

Reply:
The changes were tagged at a feat, not a fix. So this is doing the right thing. No?

Reply:
Or are we trying to keep python's major.minor in line with Looker's version?

Reply:
that would be true if your language SDK packages follwed SemVer. But they don't, they follow a modified version of Looker's CalVer - taking the major and minor numbers from Looker's version and only ever incrementing the patch version when ANY change (breaking or otherwise) is applied to the SDK itself (I think Looker's versioning itself bumps it's patch version based on some criteria but we don't follow that)

Before Luke left, we were talking about possibly revamping this something like
1. SDKs have a runtime Looker version/capability checker to let an author know what API functionality is available for a given sdk version
2. move SDK package versions themselves back to semver

feel free to carry the torch with JK on that one :-)

Reply:
Got it.

Reply:
I have PR https://github.com/looker-open-source/sdk-codegen/pull/1009 ready if needed.

Reply:
looks like `packages/extension-sdk-react` got the `feat` bump from #1006 so we need to move to a top level `release-as: 22.2.1` entry